### ISSUE TITLE
Uncaught TypeError: this.getComponent(...).getAbsolute is not a function

### ISSUE THREAD

**Issue by matthewtoast** [Famous/engine#298]
In version 0.6.0, when calling `.getAbsoluteSize()` on a `Node`, the following error occurs:

```
Uncaught TypeError: this.getComponent(...).getAbsolute is not a function
```

It looks like the issue is here, https://github.com/Famous/engine/blob/develop/core/Node.js#L576 , where `this.getComponent(this._sizeID).getAbsolute();` is called.

Whereas the method in the Size class is `.getAbsoluteSize()`: https://github.com/Famous/engine/blob/develop/core/Size.js#L211

**Comment by DnMllr** [comment]
Fix in pr #300

**Comment by michaelobriena** [comment]
Fix laned, closing
